tiago_trajectory_controller/src/run_traj_control.cpp

```diff
@@ -123,7 +123,7 @@ void waypoints_arm_goal(control_msgs::FollowJointTrajectoryGoal& goal)
   goal.trajectory.points[index].velocities.resize(7);
   for (int j = 0; j < 7; ++j)
   {
-    goal.trajectory.points[index].velocities[j] = 0.0;
+    goal.trajectory.points[index].velocities[j] = 1.0;
```

Comment:
This one should be kept at 0 so that the end slows down until reaching the final waypoing